### DISCUSSION TITLE
Automatically disable Explorer Patcher in case not compatible with current Windows Version

### DISCUSSION THREAD

**Discussion by shekharsom** [valinet/ExplorerPatcher#2692] [Ideas]
I have been using Explorer Patcher for about 2 years now and I love it. However, every time windows comes up with something new, explorer patcher crashes the explorer. Now it results into a very bad experience of uninstalling explorer patcher and then check for new version, and wait for new version and then installing again. 

If version numbers that are tested with the current install can be hardcoded in the version or if the feature flags could be checked before patching the explorer patcher or if there are some fail safes built int, this experience would be much better.

**Comment by pyrates999** [comment]
Explorer crashing might be this bug: https://github.com/valinet/ExplorerPatcher/issues/2218

**Comment by shekharsom** [comment]
Yes. May be. But what I am suggesting is an enhancement and not a bug report. Enhancement is building the check in the system so that the feature is disabled automatically.

**Reply by pyrates999** [reply]
The code is freely available.  You can try to implement this.

**Reply by shekharsom** [reply]
:-) I wish :-)

I don't know C or C++ and haven't developed anything like that. Further don't know where to even start looking. Don't have any knowledge about windows kernels.

So, am dependent on wizards like you who could do the miracles. :-)
